
From: Adrian Bunk <bunk@fs.tum.de>

drivers/scsi/gdth.c:622:1: warning: "__devinitdata" redefined
In file included from include/linux/moduleparam.h:4,
                 from include/linux/module.h:20,
                 from drivers/scsi/gdth.c:375:
include/linux/init.h:227:1: warning: this is the location of the 
previous definition

Signed-off-by: Adrian Bunk <bunk@fs.tum.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/scsi/gdth.c |    3 ---
 1 files changed, 3 deletions(-)

diff -puN drivers/scsi/gdth.c~scsi-gdth-kill-define-__devinitdata drivers/scsi/gdth.c
--- 25/drivers/scsi/gdth.c~scsi-gdth-kill-define-__devinitdata	Mon Aug  9 15:45:19 2004
+++ 25-akpm/drivers/scsi/gdth.c	Mon Aug  9 15:45:19 2004
@@ -618,9 +618,6 @@ static unchar gdth_direction_tab[0x100] 
 };
 
 /* __initfunc, __initdata macros */
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,0)
-#define __devinitdata
-#endif
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2,4,0)
 #define GDTH_INITFUNC(type, func)       type __init func 
 #include <linux/init.h>
_
